packages/uploadthing/src/sdk/utils.ts

```diff
@@ -61,9 +65,9 @@ export const uploadFilesInternal = async (
         }[];
       }
     | { error: string };
-
-  if (!res.ok || "error" in json) {
-    throw UploadThingError.fromResponse(res);
+  
+  if ("error" in json) {
+    throw UploadThingError.toObject(json);
```

Comment:
I don't think you want `toObject()` here. We'd like to throw an `UploadthingError`